src/data/finsupp/interval.lean

```diff
@@ -51,7 +51,7 @@ section range_Icc
 variables [has_zero α] [partial_order α] [locally_finite_order α] {f g : ι →₀ α} {i : ι} {a : α}
 
 /-- Pointwise `finset.Icc` bundled as a `finsupp`. -/
-@[simps] def range_Icc (f g : ι →₀ α) : ι →₀ finset α :=
+@[simps to_fun] def range_Icc (f g : ι →₀ α) : ι →₀ finset α :=
```

Comment:
Why do you need the `to_fun` projection? If everything is set up correctly, it should be simped to `coe`.

Reply:
It's not that I need the `to_fun` projection; it's that I want to disable the `support` projection so that I can state it with the correct decidability argument.

The [`to_fun` lemma](https://leanprover-community.github.io/mathlib_docs/data/finsupp/interval.html#finsupp.range_Icc_to_fun) already exists.

Reply:
Hmm, okay that's bad. Can you then just remove the `simps` completely and provide the `coe` lemma by hand?

Reply:
Changing that feels out of scope for this PR; the lemma names are unchanged from master.

I think it would be better to independently PR `simps` configurations for `finsupp`; if those go in before this PR then I'll change `to_fun` to `coe`; if they don't, then that PR will hit a build failure on this line and the same change can be made there.

Reply:
We can leave it like that for now.